privacy.md

```diff
@@ -0,0 +1,128 @@
+
+In the event that the Cockpit Project ever decides to expand its intended site audience to include children under the age of 13, those specific web pages will, in accordance with the requirements of the Children's Online Privacy Protection Act (COPPA), be clearly identified and provide an explicit privacy notice addressed to children under 13. In addition, The Cockpit Project will provide an appropriate mechanism to obtain parental approval, allow parents to subsequently make changes to or request removal of their children's personal information, and provide access to any other information as required by law.
+
+### About Links to Other Sites
+
+This site contains links to other sites. the Cockpit Project does not control the information collection of sites that can be reached through links from [cockpit-project.org](/). If you have questions about the data collection procedures of linked sites, please contact these sites directly.
+
+### Your Rights and Choices in the EEA
+
+Where the EU General Data Protection Regulation 2016/679 (“GDPR”) applies to the processing of your personal data, especially when you access the website from a country in the European Economic Area (“EEA”), you have the following rights, subject to some limitations, against the Cockpit Project:
+
+* The right to access your personal data; 
+* The right to rectify the personal data we hold about you;
+* The right to erase your personal data;
+* The right to restrict our use of your personal data;
+* The right to object to our use of your personal data;
+* The right to receive your personal data in a usable electronic format and transmit it to a third party (also known as the right of data portability); and
+* The right to lodge a complaint with your local data protection authority. 
+
+If you would like to exercise any of these rights, you may do so by [contacting the Cockpit Project on IRC](https://webchat.freenode.net/?channels=%23cockpit). Please understand, however, the rights enumerated above are not absolute in all cases.
+
+Where the GDPR applies, you also have the right to withdraw any consent you have given to uses of your personal data. If you wish to withdraw consent that you have previously provided to the Cockpit Project, [please contact us via IRC](https://webchat.freenode.net/?channels=%23cockpit) (#cockpit on irc.freenode.net). However, the withdrawal of consent will not affect the lawfulness of processing based on consent before its withdrawal.
+
+### How to Access, Modify or Update Your Information
+
+If you wish to remove your personal data from the Cockpit Project website, you may contact us on IRC and request that we remove this information from the Cockpit Project website. Other locations where you may have used your personal data as an identifier (e.g. GitHub comments, list postings in the archives, git history, chat rooms, and changelogs) will not be altered.
```

Comment:
Is it an option to just skip this paragraph, as it doesn't apply to us? Or reiterate instead what the preamble says?

Reply:
Also, it makes it clear here (with legalese) that we can't really change what's elsewhere.